### PR TITLE
Create @kununu/stylelint-config

### PR DESCRIPTION
**Pull request by ghost**
# @kununu/stylelint-config

> kununu config for stylelint

This package contains stylelint rules for consistent CSS (or SCSS) code across kununu's projects.

## 📦 Installation

If using npm 5+, use this shortcut:

```console
npx install-peerdeps --dev @kununu/stylelint-config
```

Alternatively you can install the peer dependencies manually like this:

```console
npm install --save-dev @kununu/stylelint-config stylelint@9.10.1 stylelint-config-rational-order@0.0.4 stylelint-config-standard@18.2.0 stylelint-order@2.2.0 stylelint-scss@3.5.4
```

## 💻 Usage

Create a `.stylelintrc` file with the following configuration:

```yaml
{
  "extends": "@kununu/stylelint-config"
}
```

See [docs](https://stylelint.io/user-guide/configuration) to find more detailed information on stylelint configuration and usage.

## ⚡️ Plugins

At kununu, [@kununu/eslint-config](https://www.npmjs.com/package/@kununu/eslint-config) and [@kununu/stylelint-config](https://www.npmjs.com/package/@kununu/stylelint-config) linters runs on every commit, but finding a lint error after push can be frustrating and waste time. In order to avoid it, editor plugins are available to warn you and validate as you code.

There's what we use and recommend:

**Atom**
- [linter-stylelint](https://atom.io/packages/linter-stylelint)

**Visual Code Studio**
- [stylelint](https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint)

### CODE REVIEW

File: packages/stylelint-config/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@kununu/stylelint-config",
+  "version": "1.0.0",
```
Comment (ghost) [inline packages/stylelint-config/package.json:3]: I'll publish it under `1.0.0-beta.0` first to test it all 😉

File: packages/stylelint-config/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# @kununu/stylelint-config
+
+> kununu shareable config for stylelint
+
+For consistent CSS code across kununu's repos.
+
+## Installation
+
+```console
+$ npm install --save-dev @kununu/stylelint-config stylelint
```
Comment (catarina-lopes-pt) [inline packages/stylelint-config/README.md]: Can we have here npx install, like eslint-config-kununu/README.md?

Comment (ghost) [reply packages/stylelint-config/README.md]: sure thing

**Review by catarina-lopes-pt — APPROVED** [review]
:+1:

**Pull request merged**